src/rust/ide/src/ide/integration.rs

```diff
@@ -289,19 +289,20 @@ impl Integration {
         }
 
         frp::extend! { network
-            eval_ editor_outs.node_added([]{analytics::remote_log(analytics::AnonymousData("node_added"))});
-            eval_ editor_outs.node_removed([]{analytics::remote_log(analytics::AnonymousData("node_removed"))});
-            eval_ editor_outs.nodes_collapsed([]{analytics::remote_log(analytics::AnonymousData("nodes_collapsed"))});
-            eval_ editor_outs.node_entered([]{analytics::remote_log(analytics::AnonymousData("node_entered"))});
-            eval_ editor_outs.node_exited([]{analytics::remote_log(analytics::AnonymousData("node_exited"))});
-            eval_ editor_outs.node_exited([]{analytics::remote_log(analytics::AnonymousData("node_exited"))});
-            eval_ editor_outs.on_edge_endpoints_set([]{analytics::remote_log(analytics::AnonymousData("edge_endpoints_set"))});
-            eval_ editor_outs.visualization_enabled([]{analytics::remote_log(analytics::AnonymousData("visualization_enabled"))});
-            eval_ editor_outs.visualization_disabled([]{analytics::remote_log(analytics::AnonymousData("visualization_disabled"))});
+            eval_ editor_outs.node_editing_started([]{analytics::remote_log(analytics::AnonymousData("editor_node_editing_started"))});
+            eval_ editor_outs.node_editing_finished([]{analytics::remote_log(analytics::AnonymousData("editor_node_editing_finished"))});
+            eval_ editor_outs.node_added([]{analytics::remote_log(analytics::AnonymousData("editor_node_added"))});
+            eval_ editor_outs.node_removed([]{analytics::remote_log(analytics::AnonymousData("editor_node_removed"))});
+            eval_ editor_outs.nodes_collapsed([]{analytics::remote_log(analytics::AnonymousData("editor_nodes_collapsed"))});
```

Comment:
This is also a request that can fail, right?

Reply:
Yes. Added another log in the graph where this is executed.